src/Plugin/GraphQL/SchemaExtension/SdlSchemaExtensionPluginBase.php

```diff
@@ -89,10 +89,14 @@ protected function loadDefinitionFile($type) {
     $id = $this->getPluginId();
     $definition = $this->getPluginDefinition();
     $module = $this->moduleHandler->getModule($definition['provider']);
-    $file = "{$module->getPath()}/graphql/{$id}.{$type}.graphqls";
+    $path = 'graphql/' . $id . '.' . $type . '.graphqls';
+    $file = $module->getPath() . '/' . $path;
 
     if (!file_exists($file)) {
-      throw new InvalidPluginDefinitionException(sprintf("Missing schema definition file at %s.", $file));
+      throw new InvalidPluginDefinitionException(
+        $id,
+        sprintf('The module "%s" needs to have a schema definition "%s" in its folder for "%s" to be valid.',
+          $module->getName(), $path, $definition['class']));
```

Comment:
let's revert changes in this file, as they are not related to entity query data producer